docs/source/conf.py

```diff
@@ -219,6 +220,16 @@
 # Output file base name for HTML help builder.
 htmlhelp_basename = 'StackStormDoc'
 
+
+def previous_version(ver):
+    # XXX: on incrementing major version, minor version counter is lost!
+    major, minor = ver.split('.')
```

Comment:
Unpacking the values into `major` and `minor` will fail though if `ver` contains 2 dots (split will return 3 values if you don't do `ver.split(',', 1)).